packages/web3/package.json

```diff
@@ -52,6 +52,9 @@
         "build": "rollup -c",
         "dev": "rollup -c -w",
         "test": "jest",
+        "docker": "docker-compose run --workdir='/home/node/packages/web3' web3 npm run",
+        "predocker": "docker-compose run web3 npm install --unsafe-perm",
```

Comment:
This will go into the docker container for service `web3` and install all deps. Because I added a `prepare` step to the root `package.json` scripts after `npm install` lerna will bootstrap and also build all packages (all in the docker container / the mounted volumes). Then the integration tests will be run in the docker container.

The only problem I just realised is that the `build` output will be written to the host. One must also create volumes for all the `/build` folders. Damn lerna!

Reply:
Ah, I think i got it, thanks. (I still getting lost in this spaghetti build process, I can't see the reason of all of this complication)

Reply:
I do not find it so spaghetti to be honest. I mean lerna is annoying and makes no sense here but other than that is usualy that one npm script triggers the other...

Reply:
I made virtual `/dist` folders for the docker container.